api/enums/pinia.MutationType.md

```diff
@@ -4,21 +4,21 @@ editLinks: false
 sidebarDepth: 3
 ---
 
-[API Documentation](../index.md) / [pinia](../modules/pinia.md) / MutationType
+[Documentación API](../index.md) / [pinia](../modules/pinia.md) / MutationType
```

Comment:
I believe that translate it as "Documentación de la API" sounds more natural. What do you think??

Reply:
You're right! Change is ready